src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,95 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int LIMIT_CAPACITY = 10;
+    private Object[] elements;
+    private int size;
+
+    public ArrayList() {
+        elements = new Object[LIMIT_CAPACITY];
+    }
+
     @Override
     public void add(T value) {
-
+        grow();
```

Comment:
```suggestion
        if (size == elements.length) {
            grow();
        }
```
better check before invoking the method grow. Because the name "grow" suggests that the array would be resized without any hesitation. It would more clear if you check the size before invoking it